lychee-lib/src/client.rs

```diff
@@ -378,6 +378,7 @@ impl ClientBuilder {
             exclude_link_local_ips: self.exclude_all_private || self.exclude_link_local_ips,
             exclude_loopback_ips: self.exclude_all_private || self.exclude_loopback_ips,
             include_mail: self.include_mail,
+            include_tel: false,
```

Comment:
Users of the lychee library use filters to drive the link check. Since we can't support phone numbers, let's remove the setting from the filters and assume it's `false`.

```suggestion
```